README.md

```diff
@@ -265,43 +270,47 @@ Use `record_snapshot` to take a snapshot and write the trace data to disk, as st
 
 > [!NOTE]
 > In snapshot mode, high-frequency runtime events may overwrite initialization trace data in memory before a snapshot is taken, potentially making the trace data unusable.
->  This is more likely to occur if sub-buffer sizes are not configured appropriately.
-> Support for configuring separate channels for initialization and runtime tracepoints is planned to address this limitation (see[#199](https://github.com/ros2/ros2_tracing/issues/199)).
+> This is more likely to occur if sub-buffer sizes are too small.
+> Support for configuring separate channels for initialization and runtime tracepoints is planned to address this limitation (see [#199](https://github.com/ros2/ros2_tracing/issues/199)).
 
 ## Dual session tracing
 
-Dual session mode solves the problem of losing initialization trace data by using two separate tracing sessions: one for initialization events in [snapshot mode](https://lttng.org/docs/v2.13/#doc-tracing-session-mode), and another normal tracing session for runtime events. This allows tracing to be started at any point without losing initialization data.
+Dual session mode solves the problem of losing initialization trace data by using two separate tracing sessions: one for initialization events in [snapshot mode](https://lttng.org/docs/v2.13/#doc-tracing-session-mode), and another normal tracing session for runtime events.
+This allows starting to record trace data at any point without losing initialization data.
```

Comment:
I would still use "actively" here to differentiate between the initialization snapshot session (which exists just in case and doesn't necessarily get recorded) and actively/intentionally starting to trace at runtime:

```suggestion
This allows starting to actively record trace data at any point without losing initialization data.
```